tests/test_server.py

```diff
@@ -17,6 +27,27 @@ def test_index(self):
         self.assertEqual(data['description'], 'This is the REST API Service for the shopcarts.')
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
 
-        
+    def test_get_shopcart(self):
```

Comment:
I think we should have separate tests for happy path and bad path.

Reply:
Agree on the separate tests